File: Fibonacci.java
import java.util.Scanner;
public class Fibonacci{
    public static void main(String[] args) {
        Scanner dato=new Scanner(System.in);
        int indice;
        System.out.println("Escribe un numero: ");
        indice=dato.nextInt();
        long fibonacci=calcularFibonacci(indice);
        System.out.println("El número de Fibonacci en el índice " +indice+ " es: " +fibonacci);
    }

    public static long calcularFibonacci(int n) {
        if (n<=1){
            return n;
        }else{
            long fiboAnterior=0;
            long fiboActual=1;
            long fibonacci=0;
            for (int i=2;i<=n;i++) {
                fibonacci=fiboAnterior+fiboActual;
                fiboAnterior=fiboActual;
                fiboActual=fibonacci;
            }
            return fibonacci;
        }
    }
}

File: Fibonacci.py
def calcular_fibonacci(n):
    if n <= 1:
        return n
    else:
        fibo_anterior = 0
        fibo_actual = 1
        fibonacci = 0
        for i in range(2, n + 1):
            fibonacci = fibo_anterior + fibo_actual
            fibo_anterior = fibo_actual
            fibo_actual = fibonacci
        return fibonacci

if __name__ == "__main__":
    indice = 20
    fibonacci = calcular_fibonacci(indice)
    print(f"El número de Fibonacci en el índice {indice} es: {fibonacci}")
